tests/sequence_labelling/preprocess_test.py

```diff
@@ -132,3 +131,24 @@ def test_should_select_features(self):
         features_length = len(preprocessor.features_indices)
         assert features_length == 1
         assert all_close(features_transformed, [[[1], [2], [3]]])
+
+    def test_serialize_to_json(self):
+        preprocessor = FeaturesPreprocessor(features_indices=[1])
```

Comment:
I can see now why the test isn't failing on `features_map_to_index`. Those values are really only properly set when fitting the preprocessor.

I would recommend fitting the word preprocessor (and with it the feature preprocessor) with some dummy data.